app/views/errorAlert.scala.html

```diff
@@ -8,8 +8,8 @@ <h3 id="alert-title">Error</h3>
         <div class="alert-text standalone-text">
             It looks like you're using NetLogo Web in standalone mode.
             <br />
-            If the above error is being caused by a missing primitive, we recommend a quick visit to
-                <a href="http://netlogoweb.org" target="_blank">NetLogoWeb.org</a>.
+            If the above error is being caused by an unknown primitive, we recommend a quick visit to
```

Comment:
I agree that unimplemented is clearer. I wanted to match the wording of the error exactly. I can go over to Tortoise and change "unknown primitive" to "unimplemented primitive", then change it to unimplemented here.